scratch-js/Sprite.mjs

```diff
@@ -95,46 +95,57 @@ class SpriteBase {
 
   set costumeNumber(number) {
     this._costumeNumber = ((number - 1) % this.costumes.length) + 1;
+    this.fireBackdropTrigger();
+  }
+
+  fireBackdropTrigger() {
+    if (this instanceof Stage) {
+      this._project.fireTrigger(Trigger.BACKDROP_CHANGED, {
+        backdrop: this.costume.name
+      });
+    }
   }
 
   set costume(costume) {
     if (typeof costume === "number") {
       this.costumeNumber = costume;
     }
     if (typeof costume === "string") {
+      const getSuffixedName = name =>
+        `${name} ${this instanceof Stage ? "backdrop" : "costume"}`;
```

Comment:
We actually decided in #45 to allow both "next costume" and "next backdrop" in both the sprite and the stage. There are pros and cons to that choice, but ultimately I think it's a good one.

If you could remove these changes from the PR, that would be awesome. 👍 (Also, as a general principle, let's try to change just one thing per PR.)